Errors when building using `go1.19.2 darwin/arm64` on macos 12.6

I cloned the repo and tried to install the command following the installation instructions. I got some errors when building the binary:
```
λ make build
go get
go build -v -o stonks
golang.org/x/sys/unix
# golang.org/x/sys/unix
/Users/volmedo/go/pkg/mod/golang.org/x/sys@v0.0.0-20190624142023-c5567b49c5d0/unix/zsyscall_darwin_arm64.go:28:3: //go:linkname must refer to declared function or variable
/Users/volmedo/go/pkg/mod/golang.org/x/sys@v0.0.0-20190624142023-c5567b49c5d0/unix/zsyscall_darwin_arm64.go:43:3: //go:linkname must refer to declared function or variable
/Users/volmedo/go/pkg/mod/golang.org/x/sys@v0.0.0-20190624142023-c5567b49c5d0/unix/zsyscall_darwin_arm64.go:59:3: //go:linkname must refer to declared function or variable
/Users/volmedo/go/pkg/mod/golang.org/x/sys@v0.0.0-20190624142023-c5567b49c5d0/unix/zsyscall_darwin_arm64.go:75:3: //go:linkname must refer to declared function or variable
/Users/volmedo/go/pkg/mod/golang.org/x/sys@v0.0.0-20190624142023-c5567b49c5d0/unix/zsyscall_darwin_arm64.go:90:3: //go:linkname must refer to declared function or variable
/Users/volmedo/go/pkg/mod/golang.org/x/sys@v0.0.0-20190624142023-c5567b49c5d0/unix/zsyscall_darwin_arm64.go:105:3: //go:linkname must refer to declared function or variable
/Users/volmedo/go/pkg/mod/golang.org/x/sys@v0.0.0-20190624142023-c5567b49c5d0/unix/zsyscall_darwin_arm64.go:121:3: //go:linkname must refer to declared function or variable
/Users/volmedo/go/pkg/mod/golang.org/x/sys@v0.0.0-20190624142023-c5567b49c5d0/unix/zsyscall_darwin_arm64.go:136:3: //go:linkname must refer to declared function or variable
/Users/volmedo/go/pkg/mod/golang.org/x/sys@v0.0.0-20190624142023-c5567b49c5d0/unix/zsyscall_darwin_arm64.go:151:3: //go:linkname must refer to declared function or variable
/Users/volmedo/go/pkg/mod/golang.org/x/sys@v0.0.0-20190624142023-c5567b49c5d0/unix/zsyscall_darwin_arm64.go:166:3: //go:linkname must refer to declared function or variable
/Users/volmedo/go/pkg/mod/golang.org/x/sys@v0.0.0-20190624142023-c5567b49c5d0/unix/zsyscall_darwin_arm64.go:166:3: too many errors
make: *** [build] Error 2
```

Upgrading the `golang.org/x/sys` package solved the issue.

--- a/go.mod
+++ b/go.mod
@@ -7,4 +7,5 @@ require (
 	github.com/shopspring/decimal v1.2.0
 	github.com/spf13/cobra v1.0.0
 	github.com/spf13/viper v1.7.0
+	golang.org/x/sys v0.0.0-20221006211917-84dc82d7e875 // indirect
 )

